sdk/botservice/Microsoft.Azure.Management.BotService/src/Generated/Models/FacebookChannelProperties.cs

```diff
@@ -33,16 +33,16 @@ public FacebookChannelProperties()
         /// Initializes a new instance of the FacebookChannelProperties class.
         /// </summary>
         /// <param name="appId">Facebook application id</param>
-        /// <param name="appSecret">Facebook application secret. Value only
-        /// returned through POST to the action Channel List API, otherwise
-        /// empty.</param>
         /// <param name="isEnabled">Whether this channel is enabled for the
         /// bot</param>
         /// <param name="verifyToken">Verify token. Value only returned through
         /// POST to the action Channel List API, otherwise empty.</param>
         /// <param name="pages">The list of Facebook pages</param>
+        /// <param name="appSecret">Facebook application secret. Value only
+        /// returned through POST to the action Channel List API, otherwise
+        /// empty.</param>
         /// <param name="callbackUrl">Callback Url</param>
-        public FacebookChannelProperties(string appId, string appSecret, bool isEnabled, string verifyToken = default(string), IList<FacebookPage> pages = default(IList<FacebookPage>), string callbackUrl = default(string))
+        public FacebookChannelProperties(string appId, bool isEnabled, string verifyToken = default(string), IList<FacebookPage> pages = default(IList<FacebookPage>), string appSecret = default(string), string callbackUrl = default(string))
```

Comment:
This will result in a breaking change do you have the link to the spec change associated with this?

Reply:
will add custom constructor for back compat